includes/Upgrade/Upgrades/BackgroundProcesses/V_4_3_1_RegenerateReportData.php

```
@@ -0,0 +1,27 @@
+ * Regenerate WooCommerce report data in background.
+ *
+ * @since DOKAN_SINCE
+ */
+class V_4_2_11_RegenerateReportData extends DokanBackgroundProcesses {
+
+    /**
+     * @param mixed $item
+     *
+     * @return bool
+     */
+    public function task( $item ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$item'. (undefined)

(UnusedFormalParameter)